modules/01-kafka-on-openshift.md

```diff
@@ -98,4 +98,79 @@ The `KafkaUser` resources are described in the `kafka/kafka-users.yaml` file.
 
 ```shell
 oc apply -f kafka/kafka-users.yaml
+```
+
+## Deploying resources for visualizing metrics
+
+To visualize metrics from Kafka, Kafka Connect and Zookeeper containers we need to deploy visualization chain. The first segment is Prometheus Operator by CoreOS.
+This operator allows us to easily deploy another segment - Prometheus server. With Prometheus server deployed we will need the last segment - Grafana.
+Grafana is a graphical tool which displays selected metrics very easily.
+
+### Deploying Prometheus Operator
+To deploy Prometheus Operator we can use single file. 
```

Comment:
there was a step about changing the namespace (other then removing security context here?). Did you set a fixed namespace in the YAML? what if people will want to use a different one when they redo the demo at home?